content/post/practical-tb.Rmd

```diff
@@ -0,0 +1,889 @@
+relapse  <-                 # Relapse rate
+  
+```
+```{r include=FALSE}
+
+# Model Parameters
+T.lfx    <- 72              # Life expectancy
+T.dur    <- 3               # Duration of infectious period
+beta     <- 5             # Transmission rate per capita
+break_in <- 0.1*(1/T.lfx)   # Transition rate from latent into active disease
+selfcure <- 0.5*(1/T.dur)   # Rate of spontaneous cure
+mu       <- 1/T.lfx         # Background mortality rate
+mu_tb    <- 0.5*(1/T.dur)   # TB mortality rate
+fast     <- 0.1             # Fraction fast progressing to active TB
+imm      <- 0.5             # Infectiousness decline (partial immunity)
+relapse  <- 0.005           # Relapse rate
+```
+
+
+Now we have a full list of parameters and a general idea of the flows between compartments. The code you will find below describes the  set of ordinary differential equations that describe our system. Each equation (named with prefix *d*) represent a compartment in our model with respective flows in and out, at each time step. 
+
+During the lecture the concept of recent *vs.* remote incidence was introduced. Since we want to set up a model that allows us to track our epidemic changes over time and under different interventions, we need to code model outputs for the recent and remote incidence components.
+
+**2) Look at the code below. Using the rates and stages in the first equations, code the model outputs for remote and recent TB incidence:** 
+
+*Copy and Paste the code below*
+
+```{r eval=FALSE}
+TB.Basic <- function (t, state, parameters) {
+  with(as.list(c(state,parameters)),             
+       {
+         N      <- U + L + I + R            # Total population
+         births <- I * mu_tb + N * mu       # Births (for stable population)
+         lambda <-   beta * I/N             # Force of Infection
+         
+         # Uninfected 
+         dU <- births - U * (lambda+mu)                              
+         
+         # Latent 
+         dL <-  U * lambda * (1-fast) + R * (lambda * (1-fast) * imm) - L * (mu + break_in) 
+         
+         # Active TB
+         dI <-  U * lambda * fast + R * (lambda * fast * imm) +  L * break_in + R * relapse -
+           I * (mu + mu_tb + selfcure)
+         
+         # Recovered
+         dR <-  I * selfcure - R * (lambda * imm + relapse + mu)      
+         
+         # Model outcomes
+         dIncidence <- U * (lambda * fast) + R * (lambda * fast * imm) + L * break_in + R * relapse 
+         
+         #::::::::: TRY  TO COMPLETE THIS EQUATIONS 
+         
+         dIrecent   <-
+
+         dIremote   <-
+        
+         #::::::::::::::::::::::::::::::::::::::::::::
+           
+         #wrap-up 
+         dx <- c(dU, dL, dI, dR, dIncidence, dIrecent , dIremote)
+         list(dx)
+       }
+  )
+}
+```
+
+
+
+
+**3) Looking at the code above, can you write down (*on paper*) the differential equations for this model?**  
+
+
+```{r include=FALSE}
```

Comment:
Would you like me to include a parameter for showing/hiding chunks? For example, this has already been done here: https://github.com/reconhub/learn/blob/e39d940055f8ad6bce0f29c58d6c6f67738348ce/content/post/practical-real-time-response-2.Rmd#L11-L20